.github/workflows/integration_tests.yml

```diff
@@ -14,9 +14,96 @@ on:
   
 jobs:
   
+  OVPsim_Integration_Tests:
+    name: OVPsim Integration Tests
+    runs-on: ubuntu-20.04
+    continue-on-error: true
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Install TFLM dependencies
+      run: |
+        sudo apt-get install python3-setuptools
+        pip3 install wheel
+        pip3 install Pillow
+        pip3 install Wave
+        
+    - name: Get date of last cache build
+      id: date
+      run: echo "::set-output name=date::$(date +'%Y-%m-%d' -d "last saturday")"
+        
+    - name: Download Cached GCC Binaries
+      id: cache-gcc
+      uses: actions/cache@v3
+      env:
+          cache-name: cache-gcc-binaries
+      with:
+          path: ./Toolchain
+          key: gcc-binaries-${{ steps.date.outputs.date }}
+          restore-keys: |
+            gcc-binaries-
+    
+    - name: Download OVPsim
+      working-directory: ./Sim/OVPsim/bin
+      run: ./download.sh
+      
+    - name: Download TFLM
+      working-directory: ./Integration/TFLM
+      run: git clone https://github.com/tensorflow/tflite-micro.git
+      
+    - name: Patch TFLM
+      working-directory: ./Integration/TFLM/tflite-micro
+      run: git apply ../muriscv_nn.patch
+      
+    - name: Download LLVM
+      working-directory: ./Integration/TFLM/tflite-micro
+      run: |
+        wget https://apt.llvm.org/llvm.sh
+        chmod +x llvm.sh
+        sudo ./llvm.sh 14
+        rm llvm.sh
+      
+    - name: TFLM Integration Test for Scalar muRISCV-NN with gcc with OVPsim
```

Comment:
I would prefer to define the following which would be more compact and make it easier to add new VLEN options etc.

(Syntax probably not 100% correct...)

```
jobs:
  OVPsim_Integration_Tests:
    strategy:
      matrix:
        config:
          - {vext: "OFF", pext: "OFF", mode: "RELEASE", toolchain: "gcc", arch: "rv32gc", vlen: 0, sim: "OVPSim"}
          - {vext: "OFF", pext: "OFF", mode: "RELEASE", toolchain: "llvm", arch: "rv32gc", vlen: 0, sim: "OVPSim"}
          - {vext: "OFF", pext: "ON", mode: "RELEASE", toolchain: "gcc", arch: "rv32gcp", vlen: 0, sim: "OVPSim"}
          - {vext: "ON", pext: "OFF", mode: "RELEASE", toolchain: "gcc", arch: "rv32gcv", vlen: 64, sim: "OVPSim"}
          - {vext: "ON", pext: "OFF", mode: "RELEASE", toolchain: "llvm", arch: "rv32gcv", vlen: 64, sim: "OVPSim"}
          - {vext: "ON", pext: "OFF", mode: "RELEASE", toolchain: "gcc", arch: "rv32gcv", vlen: 128, sim: "OVPSim"}
          - {vext: "ON", pext: "OFF", mode: "RELEASE", toolchain: "llvm", arch: "rv32gcv", vlen: 128, sim: "OVPSim"}
          - {vext: "ON", pext: "OFF", mode: "RELEASE", toolchain: "gcc", arch: "rv32gcv", vlen: 512, sim: "OVPSim"}
          - {vext: "ON", pext: "OFF", mode: "RELEASE", toolchain: "llvm", arch: "rv32gcv", vlen: 512, sim: "OVPSim"}
...
    steps:  
...

    - name: TFLM Integration Test
      working-directory: ./Integration/TFLM/tflite-micro
      run: ../tflm_integration_tests.sh ${{ matrix.config.vext }} ${{ matrix.config.pext }} ${{ matrix.config.mode }} ${{ matrix.config.toolchain }} ${{ matrix.config.arch }} ${{ matrix.config.vlen }} ${{ matrix.config.sim }}
```